extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -270,6 +270,9 @@ export class SubscriptionService {
           'Selected service not registered');
 
       selectedPlatform.activate(renderState);
+      if (selectedPlatform.isPingbackEnabled()) {
```

Comment:
There are also rules when pingback can be triggered. And right after authorization is not the right time. Instead, we need to implement pingback impression. I'm ok if you do it in a follow up PR, just please create a bug for it so that we don't forget. The algorithm is https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/0.1/amp-access.js#L577. I'd strongly prefer if we put it in a separate file, e.g. `ViewerTracker`. It needs to be started immediately after grant state is "known" and applied.

Reply:
Added, tests WIP